### PR TITLE
Rand r 1.2.2.6/7

### PR DESCRIPTION
**Pull request by greta-schock**

### CODE REVIEW

**Comment by greta-schock** [comment]
This branch has 1.2.2.8 in there, too

File: client/components/RatingsAndReviews/components/HelpfulReviewFeedbackButton.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import LogClick from '../../LogClick.jsx'
+import axios from 'axios';
+
+const HelpfulReviewFeedbackButton = ({ review_id, helpfulnessRating }) => {
```
Comment (bryancWA) [inline client/components/RatingsAndReviews/components/HelpfulReviewFeedbackButton.jsx:5]: I didn't understand the level to which you like long variable names... Looks good!

File: server/index.js [hunk 2]
```diff
@@ -120,6 +120,20 @@ app.get('/api/getAllReviews', (req, res) => {
     });
 });
 
+//MARK REVIEW AS HELPFUL- updates a review to show it was found helpful 
+app.put('/api/reviews/:review_id/helpful', (req, res) => {
+  //the request will have two values: the id of what to change and the +1 incremented value of the helpfulness review (on the client side)
+  const { review_id } = req.params;
+  axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/reviews/${review_id}/helpful`, {body: {review_id}}, { headers: { Authorization: API_KEY_GS}})
```
Comment (bryancWA) [inline server/index.js:127]: with your put request do you really need the review_id in the URL and as a parameter?

**Review by bryancWA — COMMENTED** [review]
Looks good. Only thing that I was curious about was the redundancy of the review_id in the put request.

File: server/index.js (hunk 2 above)
Comment (grsc0529) [reply server/index.js:127]: I'll try to reconfig!

Comment (greta-schock) [reply server/index.js:127]: If I remember correctly, the "increment" effect this PUT invokes is by giving the review_id as a part of the body. It's weird, but it works ¯\_(ツ)_/¯

**Pull request merged**